src/sentry/issues/occurrence_consumer.py

```diff
@@ -215,8 +244,10 @@ def submit(self, message: Message[KafkaPayload]) -> None:
         try:
             payload = json.loads(message.payload.value, use_rapid_json=True)
             _process_message(payload)
-        except rapidjson.JSONDecodeError:
-            logger.exception("invalid json received")
+        except (rapidjson.JSONDecodeError, InvalidEventPayloadError, EventLookupError) as ee:
+            logger.error(ee)
```

Comment:
It's better to use `logger.exception` here so that we get the stack trace. You also don't need to pass the error to it, could just have a generic message like "Failed to process payload".

Should we just combine these all into the same except statement?